test_package/conanfile.py

```diff
@@ -4,9 +4,26 @@
 from conans import ConanFile
 
 
+TEST_PACKAGE_GYP = """{
+    'targets': [
+      {
+        'target_name': 'test_package',
+        'type': 'executable',
+        'sources': [
+          'test_package.cpp',
```

Comment:
forget to commit this file?

Reply:
you have created `test_package.gyp`, right?
but what about `test_package.cpp`?

Reply:
oh, it doen't care, because i'm just testing make generator

Reply:
done!